mapswipe_workers/tests/integration/test_transfer_results.py

```diff
@@ -81,6 +86,52 @@ def test_user_not_in_postgres(self):
         result = pg_db.retr_query(sql_query)
         self.assertIsNotNone(result)
 
+    def test_additional_results_are_not_deleted(self):
```

Comment:
@laurentS this is the test that you suggested.

Reply:
I don't know enough about how this works, apologies if I'm asking dumb stuff: the initial results are apparently created in group 115, so this test does not check for group clash, right? Would it be worth adding an extra test where you have additional results in the same group 115 during the transfer process? Basically the same as this one, only changing the groupId in the extra results.

Reply:
I added another test for this. So we have two test cases now:

1. a new result for a different group in the results
2. a new result for an existing group in the results